pkg/logger/event/event.go

```diff
@@ -95,23 +100,23 @@ func Log(realm string, fn ExtractAttribute) restful.FilterFunction {
 
 		switch evt.level {
 		case logrus.FatalLevel:
-			log.Fatal(evt.message...)
+			log.Fatal()
```

Comment:
Why do you remove the message? I think we still need the message, for example for logging error message

Reply:
I am sorry , i just didn't find this field in https://accelbyte.atlassian.net/wiki/spaces/PJ/pages/206307441/Log+Standardization. If you think we still need this, I would add it

Reply:
Yes, I think we still need this. Please add them back. Thank you

Reply:
added back.